tests/subscriptions/test_subscription.py

```diff
@@ -296,6 +386,12 @@ def test(self) -> None:
             time_window_sec=10 * 60,
             resolution_sec=60,
             entity_subscription=create_entity_subscription(),
+            metadata={
+                "project_id": 1,
+                "time_window": 10 * 60,
+                "resolution": 60,
+                "query": "MATCH (events) SELECT count() AS count",
```

Comment:
Why are these existing fields duplicated here? This isn't additional metadata - it's already in the SubscriptionData

Reply:
Thanks, updated to remove duplicated fields in metadata. The data received from payload strips away `SubscriptionData` fields and leaves the extra keys. https://github.com/getsentry/snuba/pull/3384/files#diff-8e3672e3fdccd3ce17c850cd47dfe0b41f53feabd7dcf550c30077fe3f7e31fbR180-R183